roles/setup_pem/tasks/setup_pemserver.yml

```diff
@@ -0,0 +1,31 @@
+---
+
+- name: update etc_hosts based on use_hostname
+  block:
+    - import_tasks: linux_update_etc_hosts.yml
+  become: yes
+  when:
+    - use_hostname
+  delegate_to: "{{ item.value.public_ip }}"
+
+- name: set force_pem_install based on the user input
+  set_fact:
+    force_pem_install: "{{ item.value.force_pem_install }}"
+  when: item.value.force_pem_install is defined
+  register: output
+
+- name: Remove pem server based on force_pem_install
+  block:
+    - import_tasks: rm_pem_server_install.yml
+  when: force_pem_install
+  become: yes
+  delegate_to: "{{ item.value.public_ip }}"
+
+- name: Install and configure postgres on centos7 or rhel7
```

Comment:
Name shouldn't be: **Install and configure PEM server** here ?

Reply:
Yeah. Fixing it.

Reply:
Fixed

Reply:
Thanks